src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -6,4 +6,10 @@ databaseChangeLog:
   - include:
       file: db/changelog/changes/03-insert-books.yaml
   - include:
-      file: db/changelog/changes/04-create-user-roles-table.yaml
\ No newline at end of file
+      file: db/changelog/changes/04-create-user-roles-table.yaml
+  - include:
+      file: db/changelog/changes/05-create-categories-table.yaml
+  - include:
+      file: db/changelog/changes/06-insert-categories.yaml
+  - include:
+      file: db/changelog/changes/07-create-book-categories-table.yaml
```

```suggestion
      file: db/changelog/changes/07-create-book-categories-table.yaml
      
```
always leave last line empty